src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_RankSumTest.java

```diff
@@ -23,13 +23,6 @@
     public static final String RAW_DELIM = ",";
     public static final String REDUCED_DELIM = ",";
 
-    @Override
-    public Map<String, Object> annotate(final ReferenceContext ref,
```

Comment:
So if you remove this, it's going to default to the non-allele-specific version, right?  So if you run VariantAnnotator requesting an AS_RankSum annotation, you'll get the finalized form of RankSum classic?  I think that at least deserves a warning.

Reply:
Thats true, BUT if you want raw annotations you need simply instantiate the variantAnnotator in rawAnnotations mode (set the boolean flag on construction). Then it will call annotateRawData() instead. Indeed the reason for removing this method was to allow for annotate() in the event that VariantAnnotator/HaplotypeCaller want to output a vcf with an AS_Annotation.

Reply:
For the record, I feel weird about this tool default behavior.  If you ask for an annotation expecting key "AS_FS" from VariantAnnotator then A) it won't be there and B) there will be an unexpected "FS" key in the output INFO field.